liferay-echo/liferay-portal
# LPD-18022 Implement the "move to" action and display it in the folder's ellipsis icon

## Motivation

[Link to LPD-18022](https://liferay.atlassian.net/browse/LPD-18022)
[Link to LPD-17945](https://liferay.atlassian.net/browse/LPD-17945)

### Changes

Add a new view to the Item Selector and Implement the move to action and display it in the folder's ellipsis icon.

## Test scenario 1

1. Go to Design > Page Templates > Display Page Templates
2. Create some folders
3.  Check the move action from the folder's ellipsis menu

## Results

https://github.com/liferay-echo/liferay-portal/assets/93203423/95d0185d-54ef-4168-a032-aa6da378e478
